docs/03-tutorials/01-debug-function-w-CLI.md

```diff
@@ -1,10 +1,7 @@
 ---
```

Comment:
This seemst to be duplicate with the doc `Serverless` folder: 03-tutorials/00-serverless/svls-05-debug-function/

Reply:
I'm not sure how it got there. I'll remove it.

Reply:
It was in the Getting Started guides for some reason so I moved it out of there, but I've had this PR open for a while, so maybe someone else put it in Serverless in the meantime and we ended up with two copies.